docs/react-wiki-archive/Contributing/Change-Files.md

```diff
@@ -0,0 +1,45 @@
+## Need for change files
```

Comment:
this should be re-worked/moved to v9 docs

Reply:
Make sure this is covered by existing docs, cross reference info well.

Reply:
This is covered by an existing doc, here:

https://github.com/microsoft/fluentui/blob/e0ee62f419aac133f980bc007589e6bc98d70a86/docs/react-v9/contributing/dev-workflow.md?plain=1#L88